### PR TITLE
Explore the data - distribution labels

### PR DESCRIPTION
**Pull request by TahiriNadia**
Fashion MNIST dataset is an example of a balanced dataset.
1) Train set: 60000 images/label
2) Test set: 10000 images/label

### CODE REVIEW

File: site/en/r2/tutorials/keras/basic_classification.ipynb [hunk 1]
```diff
@@ -431,7 +487,7 @@
         "id": "Wz7l27Lz9S1P"
       },
       "source": [
-        "We scale these values to a range of 0 to 1 before feeding them to the neural network model. To do so, we divide the values by 255. It's important that the *training set* and the *testing set* be preprocessed in the same way:"
+        "We scale these values to a range of 0 to 1 before feeding them to the neural network model. To do so, we divide the values by 255 (max values: treain and test). It's important that the *training set* and the *testing set* be preprocessed in the same way:"
```
Comment (lamberta) [inline site/en/r2/tutorials/keras/basic_classification.ipynb]:
```suggestion
        "Scale these values to a range of 0 to 1 before feeding them to the neural network model. To do so, divide the values by 255 (max values: train and test). It's important that the *training set* and the *testing set* are preprocessed the same way:"
```

File: site/en/r2/tutorials/keras/basic_classification.ipynb [hunk 2]
```diff
@@ -349,6 +349,34 @@
...
       ]
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "cIAcvQqMCaWf_"
+      },
+      "source": [
+        "Fashion MNIST dataset is an example of a balanced dataset with 6000 images by label (TRAIN)."
```
Comment (yashk2810) [inline site/en/r2/tutorials/keras/basic_classification.ipynb]: Fashion mnist has 60000 train images and 10000 test image (https://github.com/zalandoresearch/fashion-mnist)

Comment (TahiriNadia) [reply site/en/r2/tutorials/keras/basic_classification.ipynb]: Thank you @yashk2810 
Nice catch, effectively fashion mnist has 60,000 train images and 10,000 test image 
> len(train_labels)
60000
> len(test_labels)
10000

**Comment by TahiriNadia** [comment]
Thanks for your help and your contribution!
